policy/centos8/rke2.fc

```diff
@@ -1,13 +1,22 @@
 # vim: sw=8:ts=8:et
+
 /etc/systemd/system/rke2.*                                          --  gen_context(system_u:object_r:container_unit_file_t,s0)
 /lib/systemd/system/rke2.*                                          --  gen_context(system_u:object_r:container_unit_file_t,s0)
 /usr/local/lib/systemd/system/rke2.*                                --  gen_context(system_u:object_r:container_unit_file_t,s0)
 /usr/bin/rke2                                                       --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/local/bin/rke2                                                 --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
+#/var/lib/cni(/.*)?                                                      gen_context(system_u:object_r:container_var_lib_t,s0)
```

Comment:
Is there a reason why these are commented?

Reply:
Yep, as discussed but to record here for posterity: line-for-line parity when comparing policy between distribution targets (el7 :left_right_arrow: el8 :left_right_arrow: sle)

Reply:
Also, added comment at top of file stating this, as discussed